.gitignore

```diff
@@ -27,3 +27,5 @@ kern*.py
 benchmark-*.json
 perf.csv
 perf.svg
+# Files created by OSS-Fuzz when running locally
+fuzz_*.pkg.spec
```

Comment:
Do we also want to ignore the .dict files that seem to be produced in the bash scripts?